src/content/energy-consumption/index.md

```diff
@@ -7,61 +7,44 @@ sidebar: true
+Since its inception, Ethereum has aimed to implement a proof-of-stake consensus mechanism, but doing this without sacrificing security and decentralization has taken years of focused research and development. Therefore, the network started by using proof-of-work consensus. Proof-of-work consensus requires miners to use their computing hardware to solve a puzzle. The solution to the puzzle proves that some energy has been expended by the miner, demonstrating that they have invested real-world value for the right to add to the blockchain. Proof-of-work is just a mechanism to decide who gets to add the next block, as is proof-of-stake. Swapping proof-of-work for proof-of-stake, where the real world value invested comes from ETH staked directly in a smart contract, removes the need for miners to burn energy to add to the blockchain. Therefore, the environmental cost of securing the network is drastically reduced.
 
 ## The Merge {#the-merge}
 
-There is a functional proof-of-stake chain called the [Beacon Chain](/upgrades/beacon-chain/) that has been running since December 2020 that is demonstrating the viability of the proof-of-stake protocol. The Merge refers to the point in time when Ethereum leaves proof-of-work behind and fully adopts proof-of-stake. The Merge is expected to happen ~Q3/Q4 2022. [More on The Merge](/upgrades/merge/).
-
-## Proof-of-stake energy expenditure {#proof-of-stake-energy}
+A blockchain known as the "Beacon Chain" has been running the consensus algorithm since November 2020 alongside the PoW Ethereum Mainnet. In 2022, several proof-of-work devnets were launched and transitioned to proof-of-stake by merging with their own Beacon Chains. These helped client teams test software before moving to longer-lived networks. After these devnets, Ethereum developers launched new testnets for the community to use (Kiln & Kintsugi) and ran multiple shadow forks of existing testnets and mainnet. Now that these have stabilized, teams are moving to the final stages of testing: transitioning long-lived testnets (Ropsten, Goerli, Sepolia) to proof-of-stake. Merging Ethereum Mainnet with the Beacon Chain is expected to happen in the second half of 2022. At the moment of the merge, Ethereum's PoW mining will be switched off, PoS consensus will take over, and the energy consumed by the network will drop to <0.05% of its pre-merge amount.
```

Comment:
```suggestion
[The Beacon Chain](/upgrades/beacon-chain/) has been running the proof-of-stake since November 2020 alongside the proof-of-work Ethereum Mainnet. In 2022, Ethereum developers transitioned several **testing networks (testnets)** running proof-of-stake to proof-of-stake by merging with their own Beacon Chains. These helped client teams test the software before moving to longer-lived networks. After these testnets, Ethereum developers launched new testnets for the community to use (Kiln & Kintsugi) and ran multiple shadow forks of existing testnets and Mainnet. Now that these have stabilized, teams are moving to the final stages of testing: transitioning long-lived testnets (Ropsten, Goerli, Sepolia) to proof-of-stake. Merging Ethereum Mainnet with the Beacon Chain is expected to happen in the second half of 2022. At the moment of the merge, Ethereum's proof-of-work mining will be switched off, proof-of-stake consensus will take over, and the energy consumed by the network will drop to <0.05% of its pre-merge amount.
```